frontend/src/Pages/AdminManagement.tsx

```diff
@@ -176,65 +177,78 @@ export default function AdminManagement() {
                         data-tip="Add Administrator"
                     >
                         <button
-                            className="btn btn-primary btn-sm"
+                            className="btn btn-primary btn-sm text-white"
                             onClick={() => addUserModal.current?.showModal()}
                         >
-                            <UserPlusIcon className="h-4" />
+                            <PlusIcon className="h-4 border border-white rounded-full p-1 mr-2" />{' '}
+                            {/* Circular border around PlusIcon */}
+                            Add Admin
                         </button>
                     </div>
                 </div>
-                <table className="table">
+                <table className="table-2 w-full border-separate border-spacing-y-4">
                     <thead>
-                        <tr className="border-gray-600">
-                            <th className="flex">
-                                <span>Name</span>
+                        <tr>
+                            <th className="px-6 pb-2 text-left text-sm font-medium text-gray-1">
+                                Name
+                            </th>
+                            <th className="px-6 pb-2 text-left text-sm font-medium text-gray-1">
+                                Username
+                            </th>
+                            <th className="px-6 pb-2 text-left text-sm font-medium text-gray-1">
+                                Last Active
+                            </th>
+                            <th className="px-6 pb-2 text-left text-sm font-medium text-gray-1">
+                                Actions
```

Comment:
I don't think you'll need to have extra classes here, they should be pre-set if you use `table-2`. A similar table was implemented on `ProviderPlatformManagement.tsx` around line 153. It should look fairly similar to that.